examples/OAuth Samples/node_oauth/index.js

```diff
@@ -0,0 +1,92 @@
+
+    if (req.query.code) {
+
+        // #3: 
+        // ---> Request `access_token` and `refresh_token` pair by making a request to Miro /oauth endpoint. 
+        // ---> Required parameters include `grant_type`, `client_id`, `client_secret`, `code`, and `redirect_uri`.
+        // ---> See full details in Miro documentation here: https://beta.developers.miro.com/docs/getting-started-with-oauth#step-3
+
+        var url = 'https://api.miro.com/v1/oauth/token?grant_type=authorization_code&client_id=' + process.env.clientID + '&client_secret=' + process.env.clientSecret + '&code=' + req.query.code + '&redirect_uri=' + process.env.redirectURL
+    
+        axios.post(url, {}).then(function(response) {
+            console.log(`access_token: ${response.data.access_token}`)
+            console.log(`refresh_token: ${response.data.refresh_token}`)
+
+            // Set global variable for access_token value
+            const access_token = response.data.access_token;
+
+            // Specify Miro API request URL
+            var requestUrl = `https://api.miro.com/v2/boards/${process.env.boardId}`
+            
+            // #4:
+            // ---> If `access_token` was successfully retrieved, send an API request to any Miro endpoint that contains the same permissions as your OAuth 2.0 app, with `access_token` as value for Bearer Token. 
+            // ---> (See permissions under user profile > apps: https://miro.com/app/settings/user-profile/apps)
+
+            if (access_token) {
+                // API request options
+                var config = {
+                    method: 'get',
+                    url: requestUrl,
+                    headers: { 
+                      'Authorization': `Bearer ${access_token}`
+                    }
+                  }
+                  axios(config)
+                  .then(function (response) {
+                    console.log(JSON.stringify(response.data))
+                    
+                    // Create local variable for API response
+                    var miroResponse = JSON.stringify(response.data)
+
+                    // Display response in browser
+                        var JSONResponse = `<pre><code>${miroResponse}</code></pre>`
+                        res.send(`
+                            <div class="container">
+                                <h1>Hello, World!</h1>
+                                <h3>Miro API Response:</h3>
+                                ${JSONResponse}
+                            </div>
+                        `)
+                  })
+                  .catch(function (error) {
+                    console.log(error)
+                  })
+                }  
+          }).catch(function(error) {
+            console.log(error)
+        })
```

Comment:
To avoid all of this nesting, we could replace `.then()` with `async`/`await` & `try`/`catch` might make it easier to read & follow.

Reply:
Will do!